src/commands/Dev/test/appName.test.ts

```diff
@@ -0,0 +1,116 @@
+import { describe, expect, jest, test } from '@jest/globals';
+import { program, Command } from 'commander';
+import { readFileSync, writeFileSync } from 'jsonfile';
+
+import devCommand, { devCommandHandle } from '../devCommand';
+import {
+  APP_NAME,
+  DIR_BUILD_PATH,
+  WEBPACK_CONTENT
+} from '../../../../unit_test/constant';
+import { WRITE_FILE_OPTIONS } from '../../../constant';
+
+import {
+  authCommandImplement,
+  createTemplate,
+  getRandomProjectName,
+  initProject
+} from '../../../../unit_test/helper';
+import {
+  dataInitDevCommand,
+  dataTest,
+  devCommandInit
+} from './uploadConfig.test';
+
+const initializeTestProject = async () => {
+  const projectName = getRandomProjectName();
+  const currentDir = `${DIR_BUILD_PATH}/${projectName}/${APP_NAME}`;
+  const webpackDir = `${currentDir}/webpack.config.js`;
+
+  await initProject(DIR_BUILD_PATH, projectName);
+  await createTemplate(DIR_BUILD_PATH, projectName);
+
+  return webpackDir;
+};
+
+describe('Dev command', () => {
+  describe('App name', () => {
+    test('Should be "test-app" when setting the value "test-app" to --app-name option', async () => {
+      const options = ['node', 'dev', '--app-name', APP_NAME];
+      const webpackDir = await initializeTestProject();
+      authCommandImplement(program, process);
+      process.argv = options;
+      const mainProgram = devCommand(program);
+      writeFileSync(webpackDir, WEBPACK_CONTENT);
+      await mainProgram.parseAsync(process.argv);
+
+      expect(mainProgram.opts().appName).toEqual('test-app');
+    });
+
+    test('Should be "" when setting the value "" to --app-name option', async () => {
+      const options = ['node', 'dev', '--app-name', ''];
+      const webpackDir = await initializeTestProject();
+      const mainProgram = devCommand(program);
+      process.argv = options;
+      writeFileSync(webpackDir, WEBPACK_CONTENT);
+      await mainProgram.parseAsync(process.argv);
+
+      expect(mainProgram.opts().appName).toEqual('');
+    });
+
+    let mainProgram: Command;
+    const readLineAsyncParam = jest.fn();
+
+    test('Should be "" when setting the value watch to false', async () => {
```

Comment:
I found some test cases that indicate watch option test.
Is it ok in appName.test.ts?

Reply:
I created file for watch unit test:
https://github.com/kintone-labs/kintone-cli/pull/147/commits/9ef85a5d0685ea14b6cabb477010b523ec80421c

Reply:
I removed the watch with true option because when run test, the --watch in command line cause the program to get stuck, unable to run anymore:
https://github.com/kintone-labs/kintone-cli/pull/147/commits/227b40efaa40fa6d48138485796ab5158897da4e

Reply:
> I removed the watch with true option because when run test, the --watch in command line cause the program to get stuck, unable to run anymore:

Do you mean that it's no problem with behavior but we cannot run the testing command if we use the --watch option?
I think it will be better to add memo note on source code and add them later on.

Reply:
>> Do you mean that it's no problem with behavior but we cannot run the testing command if we use the --watch option?
- Yes, currently, I have not found a way to test for this situation. The detail of --watch option here: https://linux.die.net/man/1/watch

>> I think it will be better to add memo note on source code and add them later on.
- If we want to add them later, we can access to coverage file that will appear when implementing "npm run test", and find a code that has a red background, These codes have not yet covered by the jest.
For example:
<img width="561" alt="image" src="https://github.com/kintone-labs/kintone-cli/assets/117815925/f32e9f16-ffa3-464b-8fa6-44c068637634">

Reply:
Got it.
So you are suggesting to skip this testing case implementation.
I think it's inevitable and ok. Thanks for the investigation.